README.md

```diff
@@ -47,6 +47,15 @@ Will validate TypeScript code in the project. This requires a `tsconfig.json` fi
 }
 ```
 
+### proxy
```

Comment:
As noted in [the review](https://github.com/Automattic/newspack-plugin/pull/3181#pullrequestreview-2124909542) on `newspack-plugin` – these did not work in my testing.

Reply:
Deprecated the `proxy` script in favor of individual scripts in 507b877.

Reply:
Removed this docs section in 721271057800bcd1bf251995ffe414f530114ba7